code/modules/reagents/chemistry/holder.dm

```diff
@@ -597,8 +602,10 @@
 	if(!no_react)
 		target_holder.handle_reactions()
 		src.handle_reactions()
+	var/final_total = round(total_transfered_amount, CHEMICAL_QUANTISATION_LEVEL)
```

Comment:
Use CHEMICAL_VOLUME_ROUNDING. We only round reagent volumes with CHEMICAL_QUANTIZATION_LEVEL but when multiple reagent volumes are combined together the final result it rounded to CHEMICAL_VOLUME_ROUNDING.

Reply:
ah sorry, mustve been a casualty of the merge conflicts